include/ur_api_funcs.def

```diff
@@ -0,0 +1,197 @@
+
```

Comment:
Could we get a comment at the top of this saying something like "this file is autogenerated, do not edit"

Reply:
Good shout, done